deploy/README.md

```diff
@@ -14,3 +14,69 @@ The just defines the infrastructure for the express app. This includes:
   amount of pods to manage the load
 - [Config](https://kubernetes.io/docs/concepts/configuration/configmap/) - configurable values for the app
 - [Secrets](https://kubernetes.io/docs/concepts/configuration/secret/) - secret config values for the app
+
+# Alerts
+Various custom Cloud Platform alerts have been set up for the Intranet, Intranet Archive, and Justice UK.
+The following sections are short runbooks to explain what each alert means, and actions that could be taken toremedy the issue.
+
+
+# SlowResponse
+Projects: `justice-gov-uk`
+This alert fires when, over a 3 minute period, more than 1% of requests were slower than 5 seconds. In other words the p99 latency is > 5s.
+This means that some users are experiencing slow loading of the website. 
+Action could include:
+- Reviewing the ingress dashboard - the link is in the Slack message.
+- Checking the pods e.g.
+```
+NSP=intranet-dev
+2 kubectl -n $NSP get posd -owide
+```
+- Exec into a pod to identify if an fpm process has caused high CPU.
+```
+NSP=intranet-dev
+POD=$(kubectl -n $NSP get pod -l app=$NSP -o jsonpath="{.items[0].metadata.name}");
+kubectl -n $NSP exec -it $POD -c nginx -- ash
+/usr/local/bin/fpm-health/fpm-status.sh
+```
+
+# SlownessOutage
+Projects: `justice-gov-uk`
+This alert fires when, over a 3 minute period, more than 10% of requests were slower than 7.5 seconds.
+In other words the p90 latency is > 7.5s.
+This means that a significant percentage of users are experiencing slow loading of the website.
+Check the actions under the slow response section.
+Additional action could include:
+- Restart the pods with `kubectl rollout restart deployment/$NSP -n $NSP`
+
+# High404Rate
```

Comment:
This might be helpful to update to High4xxRate, with a little update about why it may be of concern.